Api/Validators/CustomRules.cs

```diff
@@ -289,6 +289,19 @@ public static IRuleBuilderOptions<T, string> IsValidName<T>(
             .WithMessage("The name must contain only letters, spaces, hyphens, apostrophes, or periods.");
     }
 
+    /// <summary>
+    /// Validates that the string is a valid city
+    /// </summary>
+    public static IRuleBuilderOptions<T, string> IsValidCity<T>(
+        this IRuleBuilder<T, string> builder)
+    {
+        return builder
+            .Must(name => name != null && name.All(
+                c => char.IsLetter(c) || c == ' ' || c == '-'))
+            .WithErrorCode(ErrorCodes.MustBeValidName)
```

Comment:
Skoro jest to inna reguła niż ta co sprawdza imiona, to powinna też mieć inny kod błędu